workflow/rocoto/gfs_tasks.py

```diff
@@ -2661,7 +2661,7 @@ def earc(self):
 
         # Integer division is floor division, but we need ceiling division
         n_groups = -(self.nmem // -self._configs['earc']['NMEM_EARCGRP'])
-        groups = ' '.join([f'{grp:02d}' for grp in range(0, n_groups)])
+        groups = ' '.join([f'{grp:02d}' for grp in range(0, n_groups+1)])
```

Comment:
For PEP-8 compliance
```suggestion
        groups = ' '.join([f'{grp:02d}' for grp in range(0, n_groups + 1)])
```